master/buildbot/steps/subunit.py

```diff
@@ -32,6 +32,11 @@ class SubunitLogObserver(logobserver.LogLineObserver, TestResult):
     parser in the most direct fashion.
     """
 
+    # XXX(sa2ajj): what are these?  I could not find anything that does anything
+    # with these?
```

Comment:
This is the only place either identifier appears, so I think it's safe to kill both.